app/experimenter/kinto/tasks/nimbus.py

```diff
@@ -0,0 +1,164 @@
+        logger.info(f"Pushing {experiment} to Kinto failed: {e}")
+        raise e
+
+
+@app.task
+@metrics.timer_decorator("check_kinto_push_queue")
+def nimbus_check_kinto_push_queue():
+    kinto_client = KintoClient(settings.KINTO_COLLECTION_NIMBUS)
+
+    metrics.incr("check_kinto_push_queue.started")
+
+    queued_experiments = NimbusExperiment.objects.filter(
+        status=NimbusExperiment.Status.REVIEW
+    )
+
+    rejected_collection_data = kinto_client.get_rejected_collection_data()
+    if rejected_collection_data:
+        rejected_slug = kinto_client.get_rejected_record()
+        experiment = NimbusExperiment.objects.get(slug=rejected_slug)
+        experiment.status = NimbusExperiment.Status.DRAFT
+        experiment.save()
+
+        generate_nimbus_changelog(
+            experiment,
+            get_kinto_user(),
+            message=f'Rejected: {rejected_collection_data["last_reviewer_comment"]}',
+        )
+
+        kinto_client.delete_rejected_record(rejected_slug)
+
+    if queued_experiments.exists():
+        if kinto_client.has_pending_review():
+            metrics.incr("check_kinto_push_queue.pending_review")
+            return
+
+        nimbus_push_experiment_to_kinto.delay(queued_experiments.first().id)
+
+        metrics.incr("check_kinto_push_queue.queued_experiment_selected")
+    else:
+        metrics.incr("check_kinto_push_queue.no_experiments_queued")
+
+    metrics.incr("check_kinto_push_queue.completed")
+
+
+@app.task
+@metrics.timer_decorator("check_experiment_is_live")
+def nimbus_check_experiment_is_live():
+    kinto_client = KintoClient(settings.KINTO_COLLECTION_NIMBUS)
+
+    metrics.incr("check_experiment_is_live.started")
+
+    accepted_experiments = NimbusExperiment.objects.filter(
+        status=NimbusExperiment.Status.ACCEPTED
+    )
+
+    records = kinto_client.get_main_records()
+    record_ids = [r.get("id") for r in records]
+
+    for experiment in accepted_experiments:
+        if experiment.slug in record_ids:
+            logger.info(
+                "{experiment} status is being updated to live".format(
+                    experiment=experiment
+                )
+            )
+
+            experiment.status = NimbusExperiment.Status.LIVE
+            experiment.save()
+
+            generate_nimbus_changelog(experiment, get_kinto_user())
+
+            logger.info("NimbusExperiment Status is set to Live")
+
+    metrics.incr("check_experiment_is_live.completed")
+
+
+@app.task
+@metrics.timer_decorator("check_experiment_is_complete")
+def nimbus_check_experiment_is_complete():
+    kinto_client = KintoClient(settings.KINTO_COLLECTION_NIMBUS)
+
+    metrics.incr("check_experiment_is_complete.started")
+
+    live_experiments = NimbusExperiment.objects.filter(
+        status=NimbusExperiment.Status.LIVE
+    )
+
+    records = kinto_client.get_main_records()
+    record_ids = [r.get("id") for r in records]
+
+    for experiment in live_experiments:
```

Comment:
Same as above regarding naming. In the absence of module docs, function doc strings, etc. the desired functioning of this process is not documented anywhere and must be inferred from the code and naming, which isn't clear here either.